03-transactions.md

```diff
@@ -395,25 +481,32 @@ committed HTLCs:
    in [Commitment Transaction](#commitment-transaction).
 1. Calculate which committed HTLCs need to be trimmed (see [Trimmed Outputs](#trimmed-outputs)).
 2. Calculate the base [commitment transaction fee](#fee-calculation).
-3. Subtract this base fee from the funder (either `to_local` or `to_remote`),
-   with a floor of 0 (see [Fee Payment](#fee-payment)).
-3. For every offered HTLC, if it is not trimmed, add an
+3. Subtract this base fee from the funder (either `to_local` or `to_remote`).
+If `option_anchor_outputs` applies to the commitment transaction,
+also subtract two times the local `dust_limit_satoshis` from the funder
+(either `to_local` or `to_remote`). Keep a floor of 0 (see [Fee Payment](#fee-payment)).
+4. For every offered HTLC, if it is not trimmed, add an
    [offered HTLC output](#offered-htlc-outputs).
-4. For every received HTLC, if it is not trimmed, add an
+5. For every received HTLC, if it is not trimmed, add an
    [received HTLC output](#received-htlc-outputs).
-5. If the `to_local` amount is greater or equal to `dust_limit_satoshis`,
+6. If the `to_local` amount is greater or equal to `dust_limit_satoshis`,
    add a [`to_local` output](#to_local-output).
-6. If the `to_remote` amount is greater or equal to `dust_limit_satoshis`,
+7. If the `to_remote` amount is greater or equal to `dust_limit_satoshis`,
    add a [`to_remote` output](#to_remote-output).
-7. Sort the outputs into [BIP 69+CLTV order](#transaction-input-and-output-ordering).
+8. If `option_anchor_outputs` applies to the commitment transaction:
+   * if `lo_local` exists and/or there are HTLCs, add a `to_local_anchor` output
+   * if `to_remote` exists and/or there are HTLCs, add a `to_remote_anchor` output
+9. Sort the outputs into [BIP 69+CLTV order](#transaction-input-and-output-ordering).
 
 # Keys
 
 ## Key Derivation
 
 Each commitment transaction uses a unique `localpubkey`, and a `remotepubkey`.
 The HTLC-success and HTLC-timeout transactions use `local_delayedpubkey` and `revocationpubkey`.
-These are changed for every transaction based on the `per_commitment_point`, with the exception of `remotepubkey` if `option_static_remotekey` is negotiated.
+These are changed for every transaction based on the `per_commitment_point`.
+For `option_static_remotekey` and `option_anchor_outputs`, no key rotation
+is applied to `remotepubkey`.
```

Comment:
Is there a reason why those two have to be bundled together? Why can't I do anchor_outputs without static_remotekey? Is there a technical reason or is it simply to slowly deprecate the non-static-remotekey mechanisms?

Reply:
My understanding is that there is no future for non-static-remotekey, so why would we want to spend time on supporting that combination?

Reply:
Resolving for now